meta/rpip-status-change-checklists.md

```diff
@@ -0,0 +1,54 @@
+
+# RPIP Status Change Checklists
+
+## Draft Checklist
+
+- [ ] Ensure that the RPIP preamble is present and correctly formatted as per RPIP-1.
+  - [ ] Add RPIP number if not present.
+  - [ ] Check `title` is a few words, and not a complete sentence.
+  - [ ] Check `author` field matches convention.
+  - [ ] Check `discussions-to` field is present, and links to the Rocket Pool forum. This field should consist of a single URL.
+  - [ ] Check `status` is set to 'Draft'
+  - [ ] Check that `type` is a valid value.
+    - [ ] If the `type` is 'protocol', ensure a `category` is present. 
+  - [ ] Check that `created` is set to the day of the RPIP number assignment, in the YYYY-MM-DD format.
+- [ ] Ensure that the RPIP matches the RPIP template in terms of layout.
+- [ ] Ensure that the RPIP author is content with the draft being published on the RPIPs portal in its current state.
+
+## Finalize Checklist
+- [ ] Ensure that the RPIP Draft Checklist has been completed for this RPIP.
+- [ ] Ensure that the RPIP preamble is present and correctly formatted for a finalized RPIP.
+  - [ ] Check `status` is set to 'Final'
+- [ ] Ensure that the finalization PR is the only open pull request modifying the RPIP in the official RPIP repository.
+- [ ] Editor Responsibilities
+  - [ ] Check that the RPIP is ready, sound and complete.
+  - [ ] Check that the ideas within the RPIP make technical sense.
+  - [ ] Check that the RPIP title accurately describes the RPIP content.
+  - [ ] Check the RPIP for language, markup, and code style.
+- [ ] Ensure that the RPIP author has confirmed the RPIP is ready to be finalized.
+- [ ] Ensure that the RPIP editors have confirmed the RPIP is ready to be finalized.
+
+## Withdrawn Checklist
+- [ ] Ensure that the RPIP author has confirmed that they are withdrawing the RPIP.
+- [ ] Check `status` is set to 'Withdrawn'
+
+## Stagnant Checklist
+- [ ] Ensure that RPIP editors are in general agreement that the RPIP has been inactive for too long.
```

Comment:
Worth adding time guidance here? I think it will be situational (some RPIPs might be drafted toward timely items), but having something like `Identify RPIPs that have no commits for the last month` might be a good starting point for identifying stagnant ones

Reply:
I think this is more a checklist for after a stagnant RPIP has been identified? I could make a general 'RPIP weekly update checklist' to contain stuff like: 'check for stagnant RPIPs'